cmd/config-seed/res/properties/edgex-support-rulesengine/application.properties

```diff
@@ -25,24 +25,17 @@ heart.beat.msg=Support Rules Engine data heart beat
 app.open.msg=This is the Support Rules Engine Microservice.
 # set port (override Spring boot default port 8080 )
 server.port=48075
-#-----------------Export Service Config----------------------------------------
-#Turn on/off registration of rules engine as export distro client.  
+#-----------------App Service Configurable Rules Config----------------------------------------
+#Turn on/off registration of rules engine as app-service-configurable client.
 #Set to false to receive messages directly from core data
 export.client=false
 expect.serializedjava=false
-export.client.registration.url=http://localhost:48071/api/v1
-#export.client.registration.url=http://edgex-export-client:48071/api/v1
-export.client.registration.name=EdgeXRulesEngine
-#use port 5566 when connected to export distro
+#use port 5566 when connected to app-service-configurable
 #use port 5563 when connected to core data directly
 export.zeromq.port=5566
 #export.zeromq.port=5563
 export.zeromq.host=tcp://localhost
-#export.zeromq.host=tcp://edgex-export-distro
-#how long to wait to retry registration
-export.client.registration.retry.time=10000
-#how many times to try registration before exiting
-export.client.registration.retry.attempts=100
+#export.zeromq.host=tcp://edgex-app-service-configurable-rules
```

Comment:
should remove this line as the configuration already in the line above and it is kind of duplicate info for it already being in docker's version as well.

Reply:
Won't fix.  This was left in intentionally and follows the established pattern of keeping docker and non-docker options in both variations of configuration files (with the non-current variation being commented out).

Reply:
ok, understanding the established pattern, but it is misleading info to left here IMHO. As configuration in the local, majority of times, it is `localhost` always